src/main/java/com/dungeoncrawler/Javiarenka/board/BoardController.java

```diff
@@ -25,6 +25,8 @@ public String fightGet(Model model) {
     public RedirectView attackHero(Hero heroFromForm, Monster monsterFromForm) {
         if (service.getSelectedHero() != null && service.getSelectedMonster() != null) {
             service.setMessageOutput(service.getMessageOutput() + "\r\n" + service.getSelectedMonster().attack(service.getSelectedHero()));
+            service.setSelectedHero(null);
+            service.setSelectedMonster(null);
         } else {
             service.setMessageOutput(service.getMessageOutput() + "\r\n" + "Hero or Monster not selected!");
```

Comment:
Jak chcemy bardziej wybajerzyć to miejsce i jeśli będziemy częściej obsługiwać takie zdarzenia, to dobrym pomysłemm byłoby napisanie do tego osobnego wyjątku i rzucenie nim w tym miejscu.